### PR TITLE
[wip] create info cmd

### PR DESCRIPTION
**Pull request by jdrouet**
Signed-off-by: Jérémie Drouet <jeremie.drouet@gmail.com>

### CODE REVIEW

File: cmd/cli/info.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+func isURL(arg string) bool {
+	_, err := url.ParseRequestURI(arg)
+	return err == nil
+}
+
+var infoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "Provides informations about a puglin",
+	Long:  "Retrieves the .reef.yml from the repository and displays the informations.",
```
Comment (ulyssessouza) [inline cmd/cli/info.go:19]: You can use `Args:  cli.ExactArgs(1),` here to avoid having the test on `len()`

File: Makefile [hunk 2]
```diff
@@ -1,11 +1,10 @@
 PLUGIN_NAME=reef
 
 GO_FLAGS=CGO_ENABLED=0
-GO_BUILD=$(GO_FLAGS) go build
```
Comment (ulyssessouza) [inline Makefile:4]: Why removing?

**Review by ulyssessouza — CHANGES_REQUESTED** [review]

File: cmd/cli/info.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 || !isURL(args[0]) {
```
Comment (ulyssessouza) [inline cmd/cli/info.go:21]: You could use directly the `_, err := url.ParseRequestURI(arg)` here and return the `error`. Using the `isURL` function you lose the error message

**Review by ulyssessouza — CHANGES_REQUESTED** [review]

File: cmd/cli/info.go (hunk 1 above)
Comment (jdrouet) [reply cmd/cli/info.go:19]: Yeah, but this way I can validate the URL

**Pull request closed without merging**